test/integration/output_test.py

```diff
@@ -2,82 +2,90 @@
+        ("out_eia__monthly_generators", 0.2, 0.02),
     ],
 )
-def test_nuclear_fraction(fast_out, df_name, expected_nuke_fraction, tolerance):
+def test_nuclear_fraction(
+    table_name: str,
+    expected_nuke_fraction: float,
+    tolerance: float,
+    pudl_engine: sa.Engine,  # Required to ensure that the data is available.
+):
     """Ensure that overall nuclear generation fractions are as expected."""
-    actual_nuke_fraction = nuke_gen_fraction(fast_out.__getattribute__(df_name)())
+    df = get_parquet_table(
+        table_name, columns=["fuel_type_code_pudl", "net_generation_mwh"]
+    )
+    actual_nuke_fraction = nuke_gen_fraction(df)
     assert abs(actual_nuke_fraction - expected_nuke_fraction) <= tolerance
 
 
 @pytest.mark.parametrize(
-    "df1_name,df2_name,mult,kwargs",
+    "test_table,mult",
     [
-        ("gens_eia860", "bga_eia860", 1 / 1, {}),
-        ("gens_eia860", "gens_eia860", 1 / 1, {}),
-        ("gens_eia860", "own_eia860", 1 / 1, {}),
-        ("gens_eia860", "plants_eia860", 1 / 1, {}),
-        ("gens_eia860", "boil_eia860", 1 / 1, {}),
-        ("gens_eia860", "pu_eia860", 1 / 1, {}),
-        ("gens_eia860", "utils_eia860", 1 / 1, {}),
-        ("gens_eia860", "bf_eia923", 12 / 1, {}),
-        ("gens_eia860", "frc_eia923", 12 / 1, {}),
-        ("gens_eia860", "gen_eia923", 12 / 1, {}),
-        ("gens_eia860", "gen_fuel_by_generator_energy_source_eia923", 12 / 1, {}),
-        ("gens_eia860", "gen_fuel_by_generator_eia923", 12 / 1, {}),
-        ("gens_eia860", "gf_eia923", 12 / 1, {}),
-        ("gens_eia860", "hr_by_unit", 12 / 1, {}),
-        ("gens_eia860", "hr_by_gen", 12 / 1, {}),
-        ("gens_eia860", "fuel_cost", 12 / 1, {}),
-        ("gens_eia860", "capacity_factor", 12 / 1, {}),
-        pytest.param(
-            "gens_eia860",
-            "mcoe_generators",
-            12 / 1,
-            {"all_gens": False},
-            marks=pytest.mark.xfail(reason="MCOE has time coverage issues."),
-        ),
+        ("core_eia860__assn_boiler_generator", 1 / 1),
+        ("_out_eia__yearly_generators", 1 / 1),
+        ("out_eia860__yearly_ownership", 1 / 1),
+        ("out_eia__yearly_plants", 1 / 1),
+        ("out_eia__yearly_boilers", 1 / 1),
+        ("_out_eia__plants_utilities", 1 / 1),
+        ("out_eia__yearly_utilities", 1 / 1),
+        ("out_eia923__monthly_boiler_fuel", 12 / 1),
+        ("out_eia923__monthly_fuel_receipts_costs", 12 / 1),
+        ("out_eia923__monthly_generation", 12 / 1),
+        ("out_eia923__monthly_generation_fuel_by_generator_energy_source", 12 / 1),
+        ("out_eia923__monthly_generation_fuel_by_generator", 12 / 1),
+        ("out_eia923__monthly_generation_fuel_combined", 12 / 1),
+        ("_out_eia__monthly_heat_rate_by_unit", 12 / 1),
+        ("_out_eia__monthly_heat_rate_by_generator", 12 / 1),
+        ("_out_eia__monthly_fuel_cost_by_generator", 12 / 1),
+        ("_out_eia__monthly_capacity_factor_by_generator", 12 / 1),
```

Comment:
- PudlTabl method names replaced with the table names they would have mapped to.
- Removed the df1 parameter since it was always the same. Ditto the kwargs.